src/content/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips.mdx

```diff
@@ -1,289 +1,291 @@
 ---
-title: Upcoming synthetic monitor public minion IP addresses
+title: Updated public minion IP address ranges
 tags:
   - Synthetics
   - Synthetic monitoring
   - Administration
 translate:
   - jp
-metaDescription: 'For New Relic synthetic minions: upcoming IP addresses to be implemented in July 2021.'
+metaDescription: 'New runtime capabilities for synthetic monitoring may require you update your firewall allow list to accept network requests from new IP address ranges'
 ---
 
-On July 15 2021, we'll be adding new IP addresses for several synthetics locations for both [US locations](#us_new_ip) and [EU locations](#eu_new_ip). To ensure your monitors are not affected by these changes, please add the appropriate IP addresses to your firewalls allow list.
+By the end of March 2022, an updated runtime will be launched to support newer versions of Chrome. As a result, you'll need to reconfigure your firewall's allow list, bot detection software, and other related applications to accept network requests from new IP address ranges. 
```

Comment:
See comment way down below, but I'd update the date as soon as you know the exact launch day.